rps.js

```diff
@@ -0,0 +1,36 @@
+const playGame = function(userChoice){
+    
+    let computerChoice = Math.floor(Math.random()*3) + 1;
```

Comment:
you can also make an array of different choices, which the computer can choose from.

doing this would make it so you can add more options inn the future.